x-pack/plugins/osquery/server/handlers/action/create_action_handler.ts

```diff
@@ -85,7 +85,7 @@ export const createActionHandler = async (
     pack_id: params.pack_id,
     pack_name: packSO?.attributes?.name,
     pack_prebuilt: params.pack_id
-      ? !!some(packSO?.references, ['type', 'osquery-pack-asset'])
+      ? some(packSO?.references, ['type', 'osquery-pack-asset'])
```

Comment:
I've been trying to remember why I put a a double bang here (I assume it was for some reason). However I don't think why some shouldnt already return boolean, so we should be good. Although - I don't think that this is an extra change that we could also avoid making in this PR (if it works, then it works:P)

Reply:
Yeah lodash `some` returns a `boolean` (`false`) if the collection is `undefined`, so no need for explicitly forcing the value to a boolean.